.github/flash_talos.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+TALOS_IP=192.168.20.9
```

Comment:
I think we do not want this hardcoded in script. 
At least we want to have a way to override the IP and user via enrivonmental variables.

Reply:
I have added input arguments, so it is now possible to override address by providing for example `flash_talos.sh user@<IP>`